index.html

```diff
@@ -161,30 +195,105 @@ <h2 class="cards-details">TONIC</h2>
+        </div>
+      </div>
+      <div class="card">
+        <img class="card-image" src="./assets/images/desktop3.png" alt="project1.png" />
+        <div class="tonic-area">
+          <h2 class="cards-details">Facebook 360</h2>
+          <div class="highlights">
+            <span class="highlight-text">FACEBOOK</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="countertext">Full Stack Dev</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="year">2015</span>
+          </div>
+          <article class="cards-description">
+            <p>Exploring the future of media in Facebook's first Virtual Reality app; 
+              a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+          </article>
+          <ul class="skills">
+            <li>html</li>
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button id="popup" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card" id="project4">
+        <img class="card-image" src="./assets/images/desktop4.png" alt="project4.png" />
+        <div class="tonic-area">
+          <h2 class="cards-details">Uber Navigation</h2>
+          <div class="highlights">
+            <span class="highlight-text">Uber</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="countertext">Lead Developer</span>
+            <img src="./assets/images/Counter.png" alt="Counter.png" />
+            <span class="year">2018</span>
+          </div>
+          <article class="cards-description">
+            <p>A smart assistant to make driving more safe, efficient, and fun by 
+              unlocking your most expensive computer: your car.
+            </p>
+          </article>
           <ul class="skills">
             <li>html</li>
+            <li>Ruby on rails</li>
             <li>css</li>
             <li>javascript</li>
           </ul>
           <button id="popup" type="button">See Project</button>
         </div>
       </div>
     </section>
+    
     <!--about-->
     <section class="about">
-      <div class="header1">
-        <h2 class="header">
+      <div class="about-mobile">
+        <h2 class="headline-text">
           About<br />
           Myself
         </h2>
-        <article class="heading-paragraph">
-          <p>
+          <p class="headline-paragraph">
             Hello I’m a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience! If
             you like what you see and have a project you need coded, don’t
             hestiate to contact me.
           </p>
-        </article>
         <div class="heading-social">
           <h2 class="icons-header">LET'S CONNECT</h2>
```

Comment:
- [ ] Kindly consider giving this tag a font-size of `16px` and a font-weight of `500` as in the figma design because we need to stick to the figma design.